skopt/space.py

```diff
@@ -23,6 +23,23 @@ def inverse_transform(self, Xt):
         return Xt
 
 
+class _Normalize:
```

Comment:
Should we rename this to maybe `HypercubeTransform` and make it public? `Normalize` can be confused with unit norm normalization.